### PR TITLE
Iotexsons Update

### PR DESCRIPTION
**Pull request by Iotexsons**
## Token Information

- **Project official website**: https://iotexsons.io

## Verification of Token

- [X] verify of your token smart contract on iotexscan.

IOS Token

Nativ: io1kz4s5gq78jvx03x9hyejlfz55rvyfrrykuyfnv

WEB3: 0xb0ab0a201e3c9867c4c5b9332fa454a0d8448c64

[https://iotexscout.io/address/io1qp6djyvndp3sksjevsw4aa7h6m79n2eyq46gra](https://iotexscout.io/address/io1kz4s5gq78jvx03x9hyejlfz55rvyfrrykuyfnv#code)

Iotexsons Avatar NFT: 

Nativ: io1vm624puwygq6x9k44ryzywqxuqu5vw6lkq8883

 
WEB3: 0x66F4aa878e2201a316D5a8c8223806e039463B5f

https://iotexscan.io/address/io1vm624puwygq6x9k44ryzywqxuqu5vw6lkq8883#transactions

 

Iotexsons Special NFT: 

Nativ: io10gdls9qf8r884dtmla40xy70c2dk9s3p6yq0r3

WEB3: 0x7a1bf8140938ce7aB57bfF6aF313Cfc29b62C221

https://iotexscan.io/address/io10gdls9qf8r884dtmla40xy70c2dk9s3p6yq0r3

- [X] add your token address to your website and list here: https://iotexsons.io/contracts

### CODE REVIEW

File: token-metadata.json [hunk 1]
```diff
@@ -644,21 +644,21 @@
...
+    "name": "Iotexsons Special",
+    "description": "Iotexsons Special NFTs",
+    "logo": "io10gdls9qf8r884dtmla40xy70c2dk9s3p6yq0r3.png",
     "type": "xrc721",
-    "symbol": "TIOSS",
-    "image_urls": ["https://iotexsons.mypinata.cloud/ipfs/QmPy7ZdRhdagFbsn28Fp9cbzTx8dsbjixPp8k4JgWWmdsq"]
+    "symbol": "IOTEXSONSSP",
+    "image_urls": ["https://iotexsons.mypinata.cloud/ipfs/QmVN86TDoUZJNgs5CM99YuhrCEKVxrqbAPuNhTmgortfys"]
   },
   "io1kz4s5gq78jvx03x9hyejlfz55rvyfrrykuyfnv": {
```
Comment (ququzone) [inline token-metadata.json:663]: If you delete this token's logo, please also remove metadata

File: token-metadata.json [hunk 2]
```diff
@@ -644,21 +644,21 @@
...
+    "image_urls": ["https://iotexsons.mypinata.cloud/ipfs/QmUsQ3B83MAEHyLSHixctzN2ZWta93ab3qpRLtzpQ6f7Rj"]
   },
-  "io1hej3flz2wqne8er0udgkl3cdzcx38frrqs7pwt": {
-    "name": "Iotexsons special",
-    "description": "The Iotexsons special collections",
-    "logo": "io1hej3flz2wqne8er0udgkl3cdzcx38frrqs7pwt.png",
+  "io10gdls9qf8r884dtmla40xy70c2dk9s3p6yq0r3": {
+    "name": "Iotexsons Special",
+    "description": "Iotexsons Special NFTs",
+    "logo": "io10gdls9qf8r884dtmla40xy70c2dk9s3p6yq0r3.png",
```
Comment (ququzone) [inline token-metadata.json:658]: Add this logo image

File: token-metadata.json (hunk 1 above)
Comment (Iotexsons) [reply token-metadata.json:679]: Hi have Corrected it

Comment (ququzone) [reply token-metadata.json:679]: Please restore `images/io1kz4s5gq78jvx03x9hyejlfz55rvyfrrykuyfnv.png` file.

Comment (Iotexsons) [reply token-metadata.json:679]: Hi, the file does exist? According to github?

**Review by ququzone — APPROVED** [review]

**Pull request merged**